src/mongocxx/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ message ("mongocxx version: ${MONGOCXX_VERSION}")
 set(MONGOCXX_INLINE_NAMESPACE "v${MONGOCXX_ABI_VERSION}")
 set(MONGOCXX_HEADER_INSTALL_DIR "${CMAKE_INSTALL_INCLUDEDIR}/mongocxx/${MONGOCXX_INLINE_NAMESPACE}" CACHE INTERNAL "")
 
-set(LIBMONGOC_REQUIRED_VERSION 1.22.1) # TODO: update to 1.24.0 once released.
+set(LIBMONGOC_REQUIRED_VERSION 1.24.0) # TODO: update to 1.24.0 once released.
```

Comment:
Aside: I expect this is not necessary for this PR. `LIBMONGOC_REQUIRED_VERSION` controls the required version of libmongoc in cmake. It does not control the version of libmongoc obtained in testing. But this seems OK to include in this PR.